backend/compute_arrivals.py

```diff
@@ -3,6 +3,7 @@
 from datetime import datetime, date, timedelta
```

Comment:
Just want to double check, did you mean to commit all this python code?

Reply:
ah 🤦 . These are leftover updates from the "download arrival data" branch.